### PR TITLE
Refactored swagger

### PR DESCRIPTION
**Pull request by satkeeva**
Signed-off-by: Ana Satkeeva <satkeeva96@gmail.com>

### CODE REVIEW

**Comment by adamgering** [comment]
I'll review soon and see if this separation makes sense.

File: rbac/server/swagger/blocks/blocks-id.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+parameters:
```
Comment (kthblmfld) [inline rbac/server/swagger/blocks/blocks-id.yaml]: Please combine all /blocks operations into one file

File: rbac/server/swagger/proposals/proposals-id.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+parameters:
```
Comment (kthblmfld) [inline rbac/server/swagger/proposals/proposals-id.yaml]: Repeat the file grouping for all endpoints. eg: One file would include

/proposals
/proposals/{proposal-id}

If it rolls up to the same url segment at the end, it should be in the same file

**Comment by satkeeva** [comment]
Closing because reopened a new one.

**Pull request closed without merging**